caim_base/models/user.py

```diff
@@ -15,5 +15,13 @@ class UserProfile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     description = models.TextField(blank=True)
     city = models.CharField(_("city"), max_length=32, blank=True)
-    state = models.CharField(_("state"), max_length=2, blank=True, null=True, choices=states.items(), default=None)
+    state = models.CharField(
+        _("state"),
```

Comment:
not a real comment on your work here, just a question: why is this way of defining field names happening? I've not seen it before

Reply:
This is actually @rwwiv 's work. I just ran Black formatter over the file before submitting the PR because I had done some work in it.

I think the underscore portion (i.e `_("state"),`) is for autotranslation of the field into other languages (in Django admin maybe?) but I had not seen that before either, (if that was the question).